app/views/admin/induction_periods/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+  page_data(
+    title: "Add induction period for #{teacher_full_name(@induction_period.teacher)}",
+    error: @induction_period.errors.present?,
+    backlink_href: admin_teacher_path(@induction_period.teacher)
+  )
+%>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <%= form_with model: [:admin, @induction_period.teacher, @induction_period], local: true do |f| %>
+      <%= content_for(:error_summary) { f.govuk_error_summary } %>
+
+      <%= f.govuk_collection_select(:appropriate_body_id,
+                                    AppropriateBody.select(:id, :name).all,
+                                    :id,
+                                    :name,
+                                    label: { text: "Which appropriate body was this induction period completed with" },
+                                    options: { include_blank: true },
+                                    class: "autocomplete"
+          ) %>
+
+      <%= f.govuk_date_field :started_on,
+          legend: { text: "Start date", size: "m" } %>
+
+      <%= f.govuk_date_field :finished_on,
+          legend: { text: "End date", size: "m" } %>
+
+      <%= f.govuk_text_field :number_of_terms,
+          label: { text: "Number of terms", size: "m" },
+          hint: { text: "Enter 0 if less than one full term, or a number with up to one decimal place (for example, 1.5)" },
+          width: 4,
```

Comment:
Since this is going to be used for integers do we know what the most common numbers will be? I would imagine it would be 1 or 2 digits. 

How would you feel about reducing the width: `width: 2`

Reply:
The scope of the ticket started with reusing the `edit` template, but then grew once we identified the addition of the AB search input. I'd say we want consistency, and presume that width was copied from the prototype. @JSJohal is this a tweaK you'd agree with?